tests/Restaurant.Tests.Integration/TableTests/CreateTableDuplicateTests.cs

```diff
@@ -0,0 +1,23 @@
+using System.Net;
+using System.Net.Http.Json;
+using Application.Dtos.Table.Requests;
+using Domain.Enums;
+using FluentAssertions;
+
+namespace Restaurant.Tests.Integration.TableTests;
+
+public class CreateTableDuplicateTests : BaseTest
```

Comment:
No, just create one file and move all tests under this. For example, TableEndpointsTests